src/libraries/Native/Unix/verify-entrypoints.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
```

Comment:
We have been discussing removal of python usage from our build system recently. So adding a new python script doesn't sound like a good thing. Since this is Unix only, a shell script would be ideal. Could you please convert this to a shell script? It should be pretty straightforward. I'll be happy to help with that if you want.

Reply:
I'd really appreciate help here. It would likely take me days to write the same in shell. :-)

Reply:
We have `CMAKE_NM` available, we can probably live even without `cmake script calling shell script` and just do the checking inside cmake script itself.

Reply:
Sure, let me put together a shell version.

Reply:
@am11 our responses have crossed. I don't think we can check that in cmake script, as it is executed at build files generation time and we need to execute it after the actual build.

Reply:
> Sure, let me put together a shell version.

Thanks! I am not using shell scripts often and this is well beyond my expertise.

TBH, I am not an expert in python either, but it is a much more approachable language, so it was my choice over bash or awk.

Reply:
@janvorli,  ok, so it is for the post-build target. We can still pass `CMAKE_NM` as argument which is located with proper mechanism for the current toolchain, rather than assuming it is available in PATH.
https://github.com/dotnet/runtime/blob/faae2249c2f15aff6a3ed5c19d0d40dc9837d5f9/eng/native/configuretools.cmake#L43

Reply:
Sure, that would make sense.